src/Types/TypeResolver.php

```diff
@@ -0,0 +1,32 @@
+<?php
+
+namespace Digia\Lumen\GraphQL\Types;
+
+use Digia\Lumen\GraphQL\Contracts\TypeResolverInterface;
+use Digia\Lumen\GraphQL\Exceptions\UnsupportedTypeException;
+use Youshido\GraphQL\Type\AbstractType;
+
+/**
+ * Class TypeResolver
+ * @package Digia\Lumen\GraphQL\Types
+ */
+class TypeResolver implements TypeResolverInterface
+{
+    /**
+     * @param array $object
+     *
+     * @return AbstractType
+     *
+     * @throws UnsupportedTypeException
+     */
+    public static function resolveType(array $object): AbstractType
```

Comment:
Not sure if it should be `resolveType(array $object)` or `resolveType(ArrayAcces $object)`

Reply:
See https://github.com/digiaonline/lumen-graphql/issues/27